tests/acceptance/features/apiOcm/share.feature

```diff
@@ -618,3 +618,33 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10285
+  Scenario: federated user upload file to a shared folder via TUS
+    Given using server "LOCAL"
+    And user "Alice" has created a folder "FOLDER" in space "Personal"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | FOLDER                        |
+      | space           | Personal                      |
+      | sharee          | Brian                         |
+      | shareType       | user                          |
+      | permissionsRole | Editor                        |
+      | federatedServer | @federation-ocis-server:10200 |
+    And using server "REMOTE"
+#    When user "Brian" lists the shares shared with him using the Graph API
+#    Then the HTTP status code should be "200"
+    And user "Brian" has created a new TUS resource in the space "Shares" with the following headers:
```

Comment:
Could you try a step something like this?
```feature
When user "Alice" uploads a file with content "lorem" to "textFile.txt" inside federated share "FOLDER" via TUS using the WebDAV API
```
The step should get the `remoteItem.id` by requesting `sharedWithMe`, and use the id to do the upload request
`remote.php/dav/spaces/{remote-item-id}/textFile.txt`
```json
{
    "createdBy": {
        "user": {
            "@libre.graph.userType": "Federated",
            "displayName": "Admin",
        }
    },
    "name": "Parent",
    "remoteItem": {
        "id": "89f37a33-858b-45fa-8890-a1f2b27d90e1$aa43c7d3-62cc-4472-b2ce-e7b14b6cff9d!Lw==",
        "name": "Parent",
    }
}
```

Reply:
@saw-jan 
Is it similiar to what the following existing step does? If yes then we can use it:
``` feature
When user "Brian" uploads a file with content "uploaded content" to "/FOLDER/file.txt" via TUS inside of the space "Shares" using the WebDAV API
```

Reply:
No, the endpoint to request will be a bit different. I referenced the new step from this one.